examples/line.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import vedo
+
+vedo.settings.default_backend = "vtk"
```

Comment:
Isn't `vtk` the default backend? Just thinking of how the example can be simplified.

```suggestion
```

Reply:
It isn't when I run the code... Without explicitly setting the VTK backend, rendering a scene doesn't do anythin on my machine.

Reply:
Strange. Didn't know that. Best keep it in then!

I've raised https://github.com/brainglobe/brainrender/issues/335 to track.